x-team/auto-ui
# Close dropdown on click by default for single-select items in SelectBox

**Release Type:** *Breaking change*

Fixes https://x-team-internal.atlassian.net/browse/XP-2426

## Description

Change the behavior of the `SelectBox` component so that the dropdown is automatically closed when used to choose only a single item.

## Checklist


## Related PRs

branch | PR
------ | ------
XP-2426-close-dropdown-after-click | [Auto](https://github.com/x-team/auto/pull/1752)

## Steps to Test or Reproduce

Play around with the SelectBox stories while looking at the code. You will notice that the default behavior now is to always close the dropdown when using it as a single-item select component. Previously, it was always necessary to include the boolean `closeDropdown` prop

## Screenshots

![selectbox](https://user-images.githubusercontent.com/7514993/50597533-13101500-0e7e-11e9-89b6-34eb8ee83b75.gif)
